### DISCUSSION TITLE
How often and when will Orca pull changes from Bambu Studio?

### DISCUSSION THREAD

**Discussion by julie777** [OrcaSlicer/OrcaSlicer#947] [Q&A]
I have been going back and forth because BS has height modifiers and Orca does not. 
However, that is not ideal, because Orca has pressure advance and I tuned my filament for it. 

I would hope that we could expect an Orca release within a couple weeks of a BS release to keep it updated with the BS features.

@SoftFever  What is the plan?

**Comment by julie777** [comment]
Part 2 - did you try to submit pull requests to BS instead of creating a new slicer?

**Reply by SoftFever** [reply]
The response I received from them earlier indicates that we have very different opinions on features and priorities. 
It's understandable, as they have a very determined goal and vision as a company.

Hence the Orca Slicer

**Reply by julie777** [reply]
> The response I received from them earlier indicates that we have very different opinions on features and priorities. It's understandable, as they have a very determined goal and vision as a company.
> 
> Hence the Orca Slicer

I get similar responses when I submit issues or feature requests. Sadly, the requests are often for the benefit of the community.

**Comment by SoftFever** [comment]
@julie777 the main branch has already merged all upstream changes.
You can download the night build here: https://github.com/SoftFever/OrcaSlicer/actions/workflows/build_win.yml

**Reply by toddanglin** [reply]
Great to know Orca Slicer is merging upstream changes/enhancements! That remains my primary concern switching from Bambu Studio. Further to @julie777's original question, is there a general goal for this project to ship upstream enhancements within a specific timeframe (like T+1 month? T+6 months?). Just curious how sync'd this project will be with the main Studio over time.

**Comment by julie777** [comment]
I have since discovered that I cannot share profiles between BS and Orca. 
Orca has settings that BS does not recognize.

This makes going back and forth harder.
